LICENSE

```diff
@@ -1,7 +1,21 @@
-Copyright 2022 Thomas A. Christensen II
+MIT License
```

Comment:
Did you confirm that changing the license is okay with the original authors? It's not something one should do lightly.

Reply:
Waiting for @MillironX, I assume since it was moved to nf-core, and I'm not sure what to do from a legal standpoint 🤷🏻‍♂️

Reply:
Yes, this code belongs to nf-core now. This is perfect.